spec/views/repo_spec.rb

```diff
@@ -39,7 +42,7 @@
 
   context 'with a last commit that has no message' do
     let(:commit) do
-      Commit.new('sha-12345', 'joe bloggs', time_days_ago(7), '')
+      Commit.build('sha-12345', 'joe bloggs', time_days_ago(7), '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.